plugins/modules/proxmox_kvm.py

```diff
@@ -270,6 +270,12 @@
       - Memory size in MB for instance.
       - This option has no default unless I(proxmox_default_behavior) is set to C(compatiblity); then the default is C(512).
     type: int
+  migrate:
+    description:
+      - Migrate the VM on I(node) if it is on another node.
```

Comment:
nit picking, maybe better as:
```suggestion
      - Migrate the VM to I(node) if it is on another node.
```
?